src/pages/Profile.js

```diff
@@ -2,13 +2,39 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import Table from 'react-bootstrap/Table';
 import ProfileMissions from '../components/missions/ProfileMissions';
+import ProfileRockets from '../components/rockets/ProfileRockets';
 import style from './Profile.module.css';
 
 export default function Profile() {
   const missionList = useSelector((state) => state.missions);
   const profileMission = missionList.filter((mission) => mission.reserved === true);
+
+  const rocketList = useSelector((state) => state.rocketReducer);
+  const profileRocket = rocketList.filter((rocket) => rocket.reservation === true);
+
```

Comment:
Great work filtering through list rockets and displaying reserved rockets in profile page ⭐ ⭐ ⭐